templates/aws/pro.html

```diff
@@ -29,18 +29,23 @@ <h1>Ubuntu Pro for AWS</h1>
 
 <section class="p-strip is-shallow u-sv3">
   <div class="row p-divider">
-    <div class="col-4 p-divider__block">
+    <div class="col-3 p-divider__block">
+      <h3 class="p-heading--4">Ubuntu Pro 22.04 LTS</h3>
+      <p>The latest LTS release includes the 5.15 kernel out of the box. Ubuntu Pro 22.04 LTS provides extended maintenance through 2032.</p>
+      <p><a href="https://aws.amazon.com/marketplace/pp/prodview-uy7jg4dds3qjw" class="p-button--positive"><span>Launch now</span></a></p>
```

Comment:
Just wondering if the text needs to be wrapped in `span`

```suggestion
      <p><a href="https://aws.amazon.com/marketplace/pp/prodview-uy7jg4dds3qjw" class="p-button--positive">Launch now</a></p>
```